sdks/common/fn-api/src/main/proto/beam_fn_api.proto

```diff
@@ -709,15 +710,19 @@ message ApiServiceDescriptor {
   // refer to this.
   string id = 1;
 
-  // (Required) The URL to connect to.
-  string url = 2;
+  // (Required) The URL to connect to. Deprecated: use payload instead
+  string url = 2 [deprecated = true];
```

Comment:
I think we want to replace the ApiServiceDescriptor here with the one you defined so just mark this entire message as deprecated saying to migrate to the definition in `endpoints.proto`. The `id` field is not actually used anywhere meaningfully.